ddtrace/logger.py

```diff
@@ -0,0 +1,33 @@
+import logging
+from logging.handlers import RotatingFileHandler
+import os
+
+
+def configure_ddtrace_logger():
+    debug_log_level = os.environ.get("DD_TRACE_DEBUG", "false").lower() in ("true", "1")
+    log_path = os.environ.get("DD_TRACE_LOG_FILE", None)
+    max_file_bytes = os.environ.get("DD_TRACE_FILE_SIZE_BYTES", 15000000)
+    num_backup = 1
+    log_format = "%(asctime)s %(levelname)s [%(name)s] [%(filename)s:%(lineno)d] {}- %(message)s"
+    log_formatter = logging.Formatter(log_format)
+
+    ddtrace_logger = logging.getLogger("ddtrace")
+    ddtrace_logger.propagate = False
+
+    if debug_log_level is False:
+        ddtrace_logger.setLevel(logging.WARN)
+    else:
+        ddtrace_logger.setLevel(logging.DEBUG)
```

Comment:
```suggestion
    ddtrace_logger.setLevel(logging.DEBUG if debug_log_level else logging.WARN)
```

you don't have to make this change, your code is absolutely fine, just showing an alternative.